src/types/Patterns.ts

```diff
@@ -39,8 +39,8 @@ const VAT_ID = {
   MX: /^[a-zA-Z\d-&]{12,13}$/,
 }
 const COMMERCIAL_REG_NUMBER = {
-  Worldwide: /^(?!.*\s$)([A-Za-z0-9](\.|\s|-)?){4,21}$/, // generic pattern
-  DE: /^(?!.*\s$)([A-Za-z0-9](\s|-)?){9}$/,
+  Worldwide: /^(?!.*\s$)([A-Za-zÀ-ÿ0-9.()](\.|\s|-)?){4,50}$/, // generic pattern
+  DE: /^(?!.*\s$)([A-Za-zÀ-ÿ0-9.()](\s|-)?){4,50}$/,
```

Comment:
see comment , does still allowe characters but this is not allowed in a HRB/HRA Number

Reply:
Hi @MaximilianHauer the name without court seems not to be unique. You can have "München HRB 123" and "Frankfurt HRB 123" afaik. Also our BPDM pool. If that is not correct we need 

There are also "VR" and some other types we should not miss if we make this very restrictive.